common/util/Serializer.h

```diff
@@ -35,7 +36,7 @@ class Serializer {
    * later be accessed with get_save_result.
    */
   Serializer() : m_writing(true) {
-    const size_t initial_size = 32;
+    constexpr size_t initial_size = 32 * 1024 * 1024;
```

Comment:
I think I have a better fix for this. There's a silly bug so it resizes on every addition :man_facepalming:

Reply:
but this is fine for now